src/main/java/bio/terra/app/utils/startup/StartupInitializer.java

```diff
@@ -11,6 +12,6 @@ private StartupInitializer() {}
 
   public static void initialize(ApplicationContext applicationContext) {
     // Initialize jobService, stairway, migrate databases, perform recovery
-    applicationContext.getBean("jobService");
+    applicationContext.getBean(JobService.class);
```

Comment:
I wonder why this is needed. Unless a bean is `@Lazy` it should be initialized when its context is initialized. See https://docs.spring.io/spring-framework/reference/core/beans/dependencies/factory-lazy-init.html

Reply:
This is a different case.  JobService is dual purpose (probably should be refactored).  Part of what it does is initialize Stairway as part of the application startup.  I've created a ticket to fix how jobservice is used (https://broadworkbench.atlassian.net/browse/DR-3129)

Reply:
OK, so this is another case where we should be using `@PostConstruct`.

Reply:
Exactly